frontend/views/containers/proposals/RemoveMember.vue

```diff
@@ -112,12 +112,12 @@ export default {
         })
         this.$refs.proposal.close()
       } catch (e) {
-        console.error(`Failed to remove member ${member}.`, e)
+        console.error('Failed to remove member %s:', member, e.message)
```

Comment:
@snowteamer Am curious why this change was made?

Reply:
@greg Because of a CodeQL alert which was blocking the PR. It was marked as "critical" although the only potential risk seems to be garbled console output.